lang/c++/test/buffertest.cc

```diff
@@ -34,19 +34,18 @@ using detail::kMinBlockSize;
 using std::cout;
 using std::endl;
 
+// Make a string of repeating 0123456789ABCDEF0123456789...
 std::string makeString(size_t len) {
-    std::string newstring;
-    newstring.reserve(len);
+    std::string result;
+    result.reserve(len);
+
+    constexpr auto chars = "0123456789ABCDEF";
```

Comment:
Not sure how `auto chars` will be resolved in this case, but I'd declare it as `constexpr auto chars[]` so that it would be definitely considered an array and any possible out-of-bounds `constexpr` access would be caught

Reply:
It's resolved to `const char*`, `constexpr auto chars[]` isn't legal, switched to `constexpr char chars[]`